app/assets/stylesheets/content/_forms.sass

```diff
@@ -543,3 +540,8 @@ fieldset.form--fieldset
 
   li, div
     padding: 0
+
+select
+  // TODO: Temporary fix, for sure it will be fixed with next foundation-app release v1.1.1
+  // remove background image after/if upgrade
+  background-image: url("data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20version%3D%221.1%22%20width%3D%2232%22%20height%3D%2224%22%20viewBox%3D%220%200%2032%2024%22%3E%3Cpolygon%20points%3D%220%2C0%2032%2C0%2016%2C24%22%20style%3D%22fill%3A%20black%22%3E%3C/polygon%3E%3C/svg%3E")
```

Comment:
@NobodysNightmare this background were already applied from foundation, I just rewrite in a way so that IE also can render this arrow...
```
media="all"
select {
  background: #fafafa url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" version="1…24"><polygon points="0,0 32,0 16,24" style="fill: black"></polygon></svg>') right 10px center no-repeat;
}
```